### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by hfsrazzimonti** [community/community#66414] [New to GitHub]
I am in Italy working for a USA company via VPN,
I cannot setup 2FA, the whole process goes as it should including sending verification code to my phone, resolved the puzzle and downloaded the recovery code txt file.
At this point I click "I saved the recovery code" and get a super generic error:
Error cannot setup 2FA authentication.

I setup my 2FA from user setting using the DUO app scanning the code and that worked.

**Comment by queenofcorgis** [comment]
Hi @hfsrazzimonti!  Thanks for joining us in the community. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
